### PR TITLE
Add information page about crash reporting

### LINKED ISSUE #3459: Move crash reporting page to docs
Looking at the first-run experience, the screen below sends Flutter users to [this wiki page](https://github.com/flutter/flutter/wiki/Flutter-CLI-crash-reporting):
image

We should move that page to the docs, as part of the general goal to not have non-contributors relying on the wiki for core experiences.

### PR DESCRIPTION
**Pull request by timsneath**
Fixes #3459

I'm proposing that this page is not included in the TOC, since its primary purpose is to be linked from the first-run experience of the Flutter CLI. But LMK if you disagree!

### CODE REVIEW

File: src/docs/reference/crash-reporting.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+---
+
+If you have not opted-out of Flutter's analytics and crash reporting, when a
+`flutter` command crashes it attempts to send a crash report to Google in order
+to help Google contribute improvements to Flutter over time. A crash report
+may contain the following information:
+
+* The name and version of your local operating system
+* The version of Flutter
+* The runtime type of the error, e.g. `StateError` or `NoSuchMethodError`
```
Comment (legalcodes) [inline src/docs/reference/crash-reporting.md]:
```suggestion
* The runtime type of the error, for example `StateError` or `NoSuchMethodError`.
```

File: src/docs/reference/crash-reporting.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+title: Flutter crash reporting
+---
+
+If you have not opted-out of Flutter's analytics and crash reporting, when a
+`flutter` command crashes it attempts to send a crash report to Google in order
+to help Google contribute improvements to Flutter over time. A crash report
+may contain the following information:
+
+* The name and version of your local operating system
```
Comment (legalcodes) [inline src/docs/reference/crash-reporting.md]:
```suggestion
* The name and version of your local operating system.
```

File: src/docs/reference/crash-reporting.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+* The version of Flutter
```
Comment (legalcodes) [inline src/docs/reference/crash-reporting.md]:
```suggestion
* The version of Flutter.
```

File: src/docs/reference/crash-reporting.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+* The runtime type of the error, e.g. `StateError` or `NoSuchMethodError`
+* The stack trace generated by the crash, which contains references to 
+  the Flutter CLI's own code and contains no references to your application
+  code
```
Comment (legalcodes) [inline src/docs/reference/crash-reporting.md]:
```suggestion
code.
```

File: src/docs/reference/crash-reporting.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  reports coming from the same computer. It also helps us verify if a fix works
+  as intended after you upgrade to the next version of Flutter.
+
+Google handles all data reported by this tool in accordance with the 
+[Google Privacy Policy][].
+
+## Disabling analytics reporting
+
+You can opt out of anonymous crash reporting and feature usage statistics from
+Flutter by running the following command.
```
Comment (legalcodes) [inline src/docs/reference/crash-reporting.md]:
```suggestion
Flutter by running the following command:
```

File: src/docs/reference/crash-reporting.md (hunk 1 above)
Comment (legalcodes) [reply src/docs/reference/crash-reporting.md]: https://developers.google.com/style/word-list#letter-i 
(see entry for avoiding using e.g.)

**Comment by timsneath** [comment]
Great, thanks! Have updated accordingly.

**Pull request merged**